client/layout/guided-tours/tours/jetpack-checklist-tour/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import { ButtonRow, makeTour, Next, Quit, Step, Tour } from 'layout/guided-tours/config-elements';
+
+export const JetpackChecklistTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			arrow="bottom-left"
+			name="init"
+			placement="above"
+			style={ {
+				marginTop: '-20px',
+			} }
+			target={ '.checklist__header-main' }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>
+						{ translate(
+							"This is your security checklist that'll help you quickly setup Jetpack. " +
+								'Pick and choose the features that you want.'
+						) }
+					</p>
+					<ButtonRow>
+						<Next step="finish">{ translate( 'Got it' ) }</Next>
+					</ButtonRow>
+				</Fragment>
+			) }
+		</Step>
+
+		<Step
+			arrow="bottom-right"
+			name="finish"
+			placement="above"
+			style={ {
+				marginTop: '-25px',
+			} }
+			target="jetpack-checklist-wpadmin-link"
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>
+						{ translate(
+							"After you're done setting everything up, you can return to your WordPress " +
+								'admin here or continue using the WordPress.com dashboard.'
```

Comment:
Might add to the WP vs WP.com confusion? :sob:

Reply:
Haha, yeah. I don't really think "WP Admin" is clear for the average user either but maybe something like  "self-hosted WordPress dashboard" as we have in the checklist footer?

![image](https://user-images.githubusercontent.com/87168/58546995-3661a500-820f-11e9-9a4b-e8ebd82f9d8a.png)

Reply:
This step's bubble is off-screen for me (13"). It'd be nice to scroll down to it. There might be a Guided Tour prop to implement that: `scrollContainer` (see https://github.com/Automattic/wp-calypso/blob/8d77bf347ec1065c9a1833dac94e549a430190ff/client/layout/guided-tours/docs/API.md#props-1)

Reply:
> Haha, yeah. I don't really think "WP Admin" is clear for the average user either but maybe something like "self-hosted WordPress dashboard" as we have in the checklist footer?
> 
> ![image](https://user-images.githubusercontent.com/87168/58546995-3661a500-820f-11e9-9a4b-e8ebd82f9d8a.png)

While `self-hosted WordPress dashboard` is more explicit, I'm still worried about mentioning the `WordPress.com dashboard` in the same sentence, this early during the setup process.

I mean it's conceivable that people making it to this step are all power users, and that they've gone through wp-admin and WP.com (for JP connect) anyway, but maybe we can still avoid mentioning `WordPress` and `WordPress.com` in the same sentence?

Maybe just refer to the `WordPress.com dashboard` as `this dashboard`? I.e. `return to your WordPress [admin panel? dashboard? installation?] or continue using this dashboard`?

Reply:
I'll switch to "this dashboard" for now, good call. I'll make a point to follow up with @jeffgolenski next week on this. I also want to revisit the "thank-you" -modal copy and the footer text all at once to make sure it's all clear and easy to understand.